app/api/crud.py

```diff
@@ -151,3 +151,44 @@ async def get(
             )
 
     return response_obj
+
+
+async def get_terms(attribute_URI: str):
+    """
+    Makes a POST request to Stardog API using httpx where the payload is a SPARQL query generated by the create_terms_query function.
+
+    Parameters
+    ----------
+    attribute_URI : str
+        Controlled term of neurobagel class for which all the available terms should be retrieved.
+
+    Returns
+    -------
+    httpx.response
+        Response of the POST request.
+    """
+    response = httpx.post(
+        url=util.QUERY_URL,
+        content=util.create_terms_query(attribute_URI),
+        headers=util.QUERY_HEADER,
+        auth=httpx.BasicAuth(
+            os.environ.get(util.GRAPH_USERNAME.name),
+            os.environ.get(util.GRAPH_PASSWORD.name),
+        ),
+    )
+
+    if not response.is_success:
+        raise HTTPException(
+            status_code=response.status_code,
+            detail=f"{response.reason_phrase}: {response.text}",
+        )
+
+    results = response.json()
+
+    results_dict = {
+        attribute_URI: [
+            i["termURL"]["value"] for i in results["results"]["bindings"]
```

Comment:
```suggestion
            result["termURL"]["value"] for result in results["results"]["bindings"]
```
:cherries: but clearer for sanity